responsive-test-utils/src/main/java/dev/responsive/kafka/api/ResponsiveTopologyTestDriver.java

```diff
@@ -81,12 +103,26 @@ private static Properties testDriverProps(
     return props;
   }
 
-  private static class CassandraClientStub extends CassandraClient {
-    CassandraClientStub() {
-      super(new ResponsiveConfig(new HashMap<>()));
+  private static class TTDMockAdmin extends MockAdminClient {
+    private static final Node BROKER = new Node(0, "dummyHost-1", 1234);
+    public TTDMockAdmin() {
+      super(Collections.singletonList(BROKER), BROKER);
     }
 
-    // TODO: override CassandraClient methods to stash data in a local in-memory stub
+    @Override
+    public DescribeTopicsResult describeTopics(Collection<String> topicNames) {
```

Comment:
are we dynamically creating topics on each `describeTopics` call? why is that necessary?

Reply:
Hahaha yeah I know, this feels very silly -- but if the topic doesn't exist then we get an `UnknownTopic` error on the `describeTopics` call in `ResponsiveConfig#getSubPartitioner`, and this was the quickest/easiest way to fix that. It didn't seem worth going beyond a hacky solution at this point